docs/platforms/javascript/common/tracing/span-metrics/index.mdx

```diff
@@ -74,42 +74,35 @@ To consistently add metrics across all spans in your application, you can use th
 
 ```javascript
 Sentry.init({
-  beforeSendTransaction(event) {
-    // Add metrics to the root span
-    event.contexts.trace.data = {
-      ...event.contexts.trace.data,
-      'app.version': '1.2.3',
-      'environment.region': 'us-west-2'
+  beforeSendSpan(span) {
+    span.data = {
+      ...span.data,
+      "app.version": "1.2.3",
+      "environment.region": "us-west-2",
     };
 
-    // Add metrics to all child spans
-    event.spans.forEach(span => {
-      span.data = {
-        ...span.data,
-        'app.component_version': '2.0.0',
-        'app.deployment_stage': 'production'
-      };
-    });
-
-    return event;
-  }
+    return span;
+  },
 });
 ```
 
 ## Best Practices for Span Metrics
 
 1. **Metric Naming**
+
```

Comment:
out of scope for this PR but as discussed yesterday, we should rethink this section.

Reply:
Yeah - im out of scope for this PR, but we're aligned that ill be removing this section.